src/BookReader/Mode1Up.js

```diff
@@ -12,387 +13,75 @@ export class Mode1Up {
-   * How much do the two bounds intersection?
-   * @param {{ top: number; bottom: number; }} bound1
-   * @param {{ top: number; bottom: number; }} bound2
-   * @returns {number}
-   */
-  static boundIntersection(bound1, bound2) {
-    const intersect = (bound1.bottom >= bound2.top) && (bound1.top <= bound2.bottom);
-    if (!intersect) return 0;
-
-    const boundingBox = {
-      top: Math.min(bound1.top, bound2.top),
-      bottom: Math.max(bound1.bottom, bound2.bottom),
-    };
-    const intersection = {
-      top: Math.max(bound1.top, bound2.top),
-      bottom: Math.min(bound1.bottom, bound2.bottom),
-    };
-    return (intersection.bottom - intersection.top) / (boundingBox.bottom - boundingBox.top);
-  }
-
-  /**
-   * Find the pages that intersect the current viewport, including 1 before/after
-   **/
-  * findIntersectingPages() {
-    // Rectangle of interest
-    const height = this.$scrollContainer.height();
-    const scrollTop = this.$scrollContainer.scrollTop();
-    const scrollBottom = scrollTop + height;
-    const scrollRegion = { top: scrollTop, bottom: scrollBottom };
-
-    let prev = null;
-    for (const {page, top, bottom} of this.pagesWithBounds()) {
-      const intersection = Mode1Up.boundIntersection({ top, bottom }, scrollRegion);
-      const cur = {page, top, bottom, intersection: intersection};
-      if (intersection) {
-        // Also yield the page just before the visible page
-        if (prev && !prev.intersection) yield prev;
-        yield cur;
-      }
-      // Also yield the page just after the last visible page
-      else if (!cur.intersection && prev?.intersection) {
-        yield cur;
-        break;
-      }
-      prev = cur;
-    }
-  }
-
-  drawLeafs() {
-    const pagesToDisplay = Array.from(this.findIntersectingPages());
-
-    if (pagesToDisplay.length) {
-      const documentContainerWidth = this.$documentContainer.width();
-
-      // The first page that's reasonably in view we set to the current index
-      const firstProperPage = (
-        pagesToDisplay.find(({intersection}) => intersection > 0.33) ||
-        pagesToDisplay[0]
-      ).page;
-      this.br.updateFirstIndex(firstProperPage.index);
-
-      for (const {page, top, bottom} of pagesToDisplay) {
-        if (!this.br.displayedIndices.includes(page.index)) {
-          const height = bottom - top;
-          const width = this.physicalInchesToDisplayPixels(page.widthInches);
-          const reduce = page.width / width;
-
-          const pageContainer = this.br._createPageContainer(page.index)
-            .update({
-              dimensions: {
-                width,
-                height,
-                top,
-                left: Math.floor((documentContainerWidth - width) / 2),
-              },
-              reduce,
-            });
-          pageContainer.$container.appendTo(this.$documentContainer);
-        }
+      .css({ overflow: 'hidden' })
+      .append(this.$el);
+
+    setTimeout(async () => {
```

Comment:
Can't remember why  I needed this in a setTimeout... I don't know why.... Might test in a future PR to see if I can remove.

I'll also check the git history; I might've left a commit message explaining.

Reply:
I REMEMBER!!!! It was because I need the element to have been appended to DOM already.

Reply:
```suggestion
    // Need this in a setTimeout so that it happens after the browser has _actually_
    // appended the element to the DOM
    setTimeout(async () => {
```

Reply:
DONE